modules/ossm-jaeger-architecture.adoc

```diff
@@ -0,0 +1,21 @@
+////
+This CONCEPT module included in the following assemblies:
+-ossm-jaeger.adoc
+////
+
+[id="ossm-jaeger-architecture_{context}"]
+= Jaeger architecture
+
+Jaeger is made up of several components that work together to collect, store, and display tracing data.
+
+* *Jaeger Client* (Tracer, Reporter, instrumented application, client libraries)- Jaeger clients are language specific implementations of the OpenTracing API. They can be used to instrument applications for distributed tracing either manually or with a variety of existing open source frameworks, such as Flask, Dropwizard, gRPC, and many more, that are already integrated with OpenTracing.
```

Comment:
This kinda gives the impression that Flask/Dropwizard/... are "natively" instrumented with OpenTracing, which isn't the case: they are instrumented by external libraries, typically hosted at the `opentracing-contrib` GitHub organization.